software/generic/LedManager.cpp

```diff
@@ -2,21 +2,22 @@
 
 #include <cassert>
 
-#include "ColorCycleEffect.hpp"
-#include "ContrastBumpsEffect.hpp"
-#include "DarkEffect.hpp"
-#include "DisplayColorPaletteEffect.hpp"
-#include "FireEffect.hpp"
-#include "FireflyEffect.hpp"
-#include "LightningEffect.hpp"
-#include "PoliceEffect.hpp"
-#include "RainbowBumpsEffect.hpp"
-#include "RainbowEffect.hpp"
-#include "RorschachEffect.hpp"
-#include "SimpleBlinkEffect.hpp"
-#include "SparkEffect.hpp"
-#include "StopLightEffect.hpp"
-#include "SwingingLights.hpp"
+#include "../../lib/effect/ColorCycleEffect.hpp"
```

Comment:
Do these need to include the `../..`? Can they be `effect/ColorCycleEffect.hpp`? If not, should we make a single `.hpp` file which includes all of the effects? This is a common strategy, I haven't really thought about whether it's good.

Reply:
Yeah, they need the prefix. I don't think that's the worst idea as long as its only the effects.